src/role-access/containers/delete-exclusion/delete-exclusion.component.ts

```diff
@@ -37,4 +41,31 @@ export class DeleteExclusionComponent implements OnInit {
       this.answers.push({label: AnswerLabelText.DateAdded, value: new Date(exclusion.added).toLocaleDateString('en-GB')});
     }
   }
+
+  public onNavEvent(navEvent: ExclusionNavigationEvent): void {
+    const goToCaseUrl = `cases/case-details/${this.caseId}/roles-and-access`;
+    switch (navEvent) {
+      case ExclusionNavigationEvent.DELETE_EXCLUSION: {
+        // might need to pass in the case id
```

Comment:
@connorpgpmcelroy - not sure whether you need the Case ID... despite the commented line, I don't see it passed anywhere?

Reply:
yes, it was passed in and then I decided against it just because I did not know whether it would be needed, will just remove comment